src/data.cpp

```diff
@@ -145,12 +145,13 @@ void InternDataHandler::getWordExamples(
 
   rslts.clear();
   for (int widx = 0; widx < doc.size(); widx++) {
+    int bc = rand() % args_->ws + 1;
```

Comment:
as we don't know the perf difference maybe make this an option, but default on rn as it's faster?

Reply:
Actually this does not affect perf that much. The speedup mostly comes from sampling a list of  negatives beforehand. I'm going to remove this change so to avoid introducing another parameter.